svo (namedb).xml

```diff
@@ -959,7 +957,7 @@ disableTrigger("NameDB qw")</script>
 					<regexCodeList>
 						<string>.*</string>
 						<string>1</string>
-						<string>^(?:Plus another (\d+) whose presence you cannot fully sense(?: \(\d+ total\))?\.|\((\d+) total\)\.)$</string>
```

Comment:
I did this change with the original issue where it was not working when there were no hidden people so this should not need to be changed.

Reply:
The changed line works both ways (tested it to verify that.) and is slightly more optimized from a regex standpoint. I removed unnecessary captures, and it doesn't have to run the entire search twice since a part is optional.

Reply:
My bad. Forgot there was a ninja change they did on me after I had kept the capture in the first place.